.github/workflows/builder.yaml

```diff
@@ -42,6 +42,7 @@ jobs:
             --batch-mode \
             --errors \
             --no-transfer-progress \
+            -Pcheckstyle \
```

Comment:
We typically do checkstyle separate in monorepo, maybe we should do the same here?

Reply:
I opt for keeping it simple now. In monorepo tests take much longer, so early feedback about checkstyle failure is important. In code samples tests are faster (a few minutes) an the changes here are much less frequent.

Reply:
Ok,  fair point